charts/gardener/operator/values.yaml

```diff
@@ -34,12 +34,15 @@ config:
   logLevel: info
   logFormat: json
   server:
-    healthProbes:
+    webhooks:
```

Comment:
Why to change the port of `healthProbes` and `metrics` ? Why not add the webhooks at port 2752?

Reply:
I consider the webhook server the "default server", so I assigned it the lowest port number.